tests/testutils/otlp_receiver_sink.go

```diff
@@ -125,14 +137,33 @@ func (otlp *OTLPReceiverSink) Start() error {
 	if err := otlp.assertBuilt("Start"); err != nil {
 		return err
 	}
-	return (*otlp.metricsReceiver).Start(context.Background(), otlp.Host)
+
+	if err := (*otlp.metricsReceiver).Start(context.Background(), otlp.Host); err != nil {
+		return err
+	}
+
+	if err := (*otlp.tracesReceiver).Start(context.Background(), otlp.Host); err != nil {
+		_ = (*otlp.metricsReceiver).Shutdown(context.Background())
+		return err
+	}
+
+	return nil
 }
 
 func (otlp *OTLPReceiverSink) Shutdown() error {
 	if err := otlp.assertBuilt("Shutdown"); err != nil {
 		return err
 	}
-	return (*otlp.metricsReceiver).Shutdown(context.Background())
+
+	metricsShutdownErr := (*otlp.metricsReceiver).Shutdown(context.Background())
+	tracesShutdownErr := (*otlp.tracesReceiver).Shutdown(context.Background())
```

Comment:
similar note as above. If they change the receiver implementation we'd need to address it but may be worth avoiding for now.

Reply:
Fixed via 4fc2a2715278b83f01b0e268ba7682e61f376f61